pkgs/development/python-modules/pytabix/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, buildPythonPackage
+, isPy3k
+, fetchPypi
+, zlib
+}:
+
+buildPythonPackage rec {
+  pname = "pytabix";
+  version = "0.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1ldp5r4ggskji6qx4bp2qxy2vrvb3fam03ksn0gq2hdxgrlg2x07";
+  };
+
+  buildInputs = [ zlib ];
+
+  doCheck = !isPy3k;
+  preCheck = ''
+    substituteInPlace test/test.py \
+      --replace 'test_remote_file' 'dont_test_remote_file'
+  '';
```

Comment:
not sure why this is still being ran for python2:
```
https://github.com/NixOS/nixpkgs/pull/98283
1 package failed to build:
python27Packages.pytabix

10 packages built:
bwa python27Packages.acebinf python37Packages.acebinf python37Packages.bwapy python37Packages.pytabix python38Packages.acebinf python38Packages.bwapy python38Packages.pytabix tebreak truvari
```

Reply:
nevermind, i read that right

Reply:
That's .. strange. What was the failure? An earlier version of this PR didn't disable the tests on py27, but this version builds for me on macos 10.14 & linux under nixpkgs-review.